ui/app/src/services/configuration.ts

```diff
@@ -51,6 +51,11 @@ interface PreviewToolsConfig {
   modules: Array<PreviewToolsModuleDescriptor>;
 }
 
+interface craftercmsModel {
+  id: string;
+
+  [prop: string]: string;
+}
```

Comment:
There's a model for this already. It's called ContentInstance (which I showed you when we discussed this stuff).